static/app/views/performance/newTraceDetails/traceRenderers/virtualizedViewManager.tsx

```diff
@@ -708,6 +708,23 @@ export class VirtualizedViewManager {
     }
   }
 
+  maybeInitializeTraceViewFromQS(fov: string): void {
```

Comment:
Only validates that the two components are float values that fall somewhere inbetween the current view. The clamping logic is handled by setTraceView. For example if we ever had x + width > entire width, the width would get clamped inside setTraceView to entire width - width